Library/Formula/slackcat.rb

```diff
@@ -0,0 +1,26 @@
+class Slackcat < Formula
+  desc "Simple command-line Utility to post snippets to Slack."
+  homepage "https://github.com/vektorlab/slackcat"
+  url "https://github.com/vektorlab/slackcat/archive/v0.7.tar.gz"
+  version "0.7"
+  sha256 "b51ab794af2a0014b5372944699d7ff9c88af4e1860206abee3a0f9bbc70d147"
+
+  depends_on "go"
+
+  def install
+    platform = `uname`.downcase.strip
+
+    unless ENV["GOPATH"]
```

Comment:
Please use our established system for handling go. You can find an example [here](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/gdm.rb#L18-L34)